### DISCUSSION TITLE
FV vs FE memory consumption

### DISCUSSION THREAD

**Discussion by heinono1** [idaholab/moose#25407] [Q&A General]
Hi,

I am testing my  FV version of the Ferret app and it seems like it consumes a LOT more memory than the FE version. Is this generally the case (or is there perhaps something goofy with my FV implementation)?

**Comment by lindsayad** [comment]
One thing you can try is setting `cache_cell_gradients = false` in your variable blocks

**Reply by heinono1** [reply]
Thanks, Alex. I'll give it a shot.
